app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,25 @@
+class SessionsController < ApplicationController
+  def new
+  end
```

Comment:
メソッドの中身がないアクション（renderのみを実行）は1行で記載すると中身の記載忘れでないことを明示できます。
```suggestion
  def new; end
```

Reply:
こんな書き方ができるのですね。大変勉強になりました。
こちらも修正いたしました。
37a2ac4